index.js

```diff
@@ -1,126 +1,19 @@
 module.exports = {
-  plugins: ['flowtype', 'import', 'prettier', 'promise', 'react', 'react-hooks'],
-  extends: ['plugin:flowtype/recommended', 'plugin:react/recommended', 'airbnb', 'prettier'],
+  plugins: [
+  ],
+  extends: [
+    'plugin:flowtype/recommended', 
+    'plugin:react/recommended', 
```

Comment:
This should belong to the `rules/react`, IMO

Reply:
I agree but for some reason extensions are not working if I move them to specific rule files, it seems the sequence is causing some rules to override (just a guess)